wiki/Full_combo/id.md

```diff
@@ -0,0 +1,16 @@
+---
+stub: true
+tags:
+  - FC
+  - full kombo
```

Comment:
might leave every instance of "full combo" untranslated here tbh, genuinely speaking i have never seen anyone referring to fc as `full kombo` or `kombo penuh` or the likes of it